### DISCUSSION TITLE
Set a theme in Windows

### DISCUSSION THREAD

**Discussion by MGuerrera** [zim-desktop-wiki/zim-desktop-wiki#2519] [Q&A]
I installed Zim on Win11 via a winget.

Now I am following this guide to install a dark theme.

[https://github.com/jaap-karssenberg/zim-wiki/wiki/Set-a-dark-theme-and-background](url)

The problem is that I can't find the "etc" folder in my program installation folder, where the "gtkrc" file should be located.

**Comment by Toggoren** [comment]
Just create it

Also you don't need to create a ``gtkrc`` file. These instructions are for older versions.


1. create folders:
    * ``C:\Program Files\Zim Desktop Wiki\share\themes``
    * ``C:\Program Files\Zim Desktop Wiki\etc\gtk-3.0\``
2. reset permissions on folders or/and "Read Only" attribute
3. put downloaded themes into ``C:\Program Files\Zim Desktop Wiki\share\themes``
4. create file ``C:\Program Files\Zim Desktop Wiki\etc\gtk-3.0\settings.ini``
    with content 
    ```ini
    [Settings]
    gtk-theme-name = Orchis-Pink-Dark
    ```
5. start ``Zim``

**Reply by MGuerrera** [reply]
Regarding point 3, you implicitly intended the creation of an additional subfolder in "etc" or meant "C: \ Program Files \ Zim Desktop Wiki \ share \ themes"?

Regarding point 2, seem I can't permanently change the permission status of etc folder: it is not a check on the read-only attribute, but an horizontal dash.

**Reply by Toggoren** [reply]
> Regarding point 3, you implicitly intended the creation of an additional subfolder in "etc" or meant "C: \ Program Files \ Zim Desktop Wiki \ share \ themes"?

Uh-huh. Аdditional subfolder in "etc"
![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/94177064/8d370ca9-f008-44e5-9624-199869d19345)
![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/94177064/8289e1d3-4739-48f4-b869-21f40f84e49b)

> Regarding point 2, seem I can't permanently change the permission status of etc folder: it is not a check on the read-only attribute, but an horizontal dash.

I didn't make this point clear enough. 

Unfortunately, I don't know the minimum required set of actions to make it working.

Personally, I gave<sup>[[instruction](https://www.uwec.edu/kb/article/drives-establishing-windows-file-and-folder-level-permissions/)]</sup> all possible permissions to all users and groups for the created folders. 

**This is redundant and reduces security, but I don't care.**

**Reply by MGuerrera** [reply]
Mmm, sorry if I insist but English is not my first language.

The first image you attached in your last reply refers to:
C:\Program Files\Zim Desktop Wiki\share\themes
and not to:
C:\Program Files\Zim Desktop Wiki\etc\share\themes

Even your point one in the first reply referred to the creation of a folder in a path that did not end in "etc" but that already appeared to me to exist in Zim folder path.

However, I also created those subfolders in "etc" and also inserted the theme there.

I tried to give all the permits as suggested by you and by the guide that I linked at the top but had no effect.

**Reply by Toggoren** [reply]
Mea culpa 🤦‍♂️

``C:\Program Files\Zim Desktop Wiki\etc\share\themes`` should be  
``C:\Program Files\Zim Desktop Wiki\share\themes``  

Corrected it in one place, forgot it in another. 🫠

The correct directory structure is on the screenshots. 👉👈

**Reply by MGuerrera** [reply]
Don't worry, it happens to get confused.

However I tried, even giving all the permissions, but when I start Zim the theme is unfortunately always white.

I would keep it even if it were just an aesthetic speech, it is that I have eye problems and all that light blinds me.
